packages/atlas/src/components/_inputs/TextField/TextField.tsx

```diff
@@ -54,8 +55,8 @@ const TextFieldComponent: React.ForwardRefRenderFunction<HTMLInputElement, TextF
           </NodeContainer>
         )}
         <TextInput
-          leftNodeWidth={nodeLeftBounds.width}
-          rightNodeWidth={nodeRightBounds.width}
+          leftNodeWidth={nodeLeftBoundsWidth}
+          rightNodeWidth={nodeRightBoundsWidth}
```

Comment:
This does not work properly, see Storybook `?path=/story/inputs-textfield--with-icons`